### PR TITLE
Updated Queue - all functionality.

### PR DESCRIPTION
**Pull request by cherryWood55**
I have made the changes as were requested.

- [x] Added the count function to count and display all the number of elements in the queue.

- [x] Added the display function to print all the elements currently in the queue.

- [x] Added a line describing the operation we are performing for every case.

Please review the changes. Thanks!

### CODE REVIEW

**Comment by cherryWood55** [comment]
I have incorporated the changes, please look into. @siddharth25pandey

**Review by siddharth25pandey — APPROVED** [review]
@cherryWood55 Looks fine to me
Able to merge.
**But my sincere advice to you is, from next time onwards do the modifications in the same PR**
**No need of creating a new PR**

**Review by Kajol-Kumari — APPROVED** [review]
Functionalities are working fine. codewise it's good to go :)

**Comment by cherryWood55** [comment]
> 
> 
> @cherryWood55 Looks fine to me
> Able to merge.
> **But my sincere advice to you is, from next time onwards do the modifications in the same PR**
> **No need of creating a new PR**

Okay. Shall keep that in mind.

File: queue_all_functionality.c [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	return new_queue;
+}
+
+int is_queue_empty(struct queue* q){
+	if (q->curr_size == 0)
+		return 1;
+	else
+		return 0;
+}
+
```
Comment (atarax665) [inline queue_all_functionality.c:38]: Add a comment above each function to explain what the function does. Everything else looks fine. :)

**Comment by cherryWood55** [comment]
@atarax665 I have added relevant comments to the code. Please review, thanks!

**Comment by cherryWood55** [comment]
Please review this PR @maintainers @iamrajiv  @atarax665 .

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@all-contributors please add @cherryWood55 for code

**Pull request merged**